sycl/source/abi_replacements_linux.txt

```diff
@@ -0,0 +1,12 @@
+_ZN2cl4sycl10level_zero10make_queueERKNS0_7contextEmb = _ZN2cl4sycl3ext6oneapi10level_zero10make_queueERKNS0_7contextEmb;
```

Comment:
Could you please describe the change to the ABI breakage check? Do we tolerate ABI breakage on library level?
We need the procedure for an update of new files.

Reply:
In short, this is a mechanism to avoid ABI breaks in our library. We have already the same for Windows (see abi_replacements_windows.cpp). If we do some ABI break, and some symbols are missing from the library and replaced with the updated ones, we can make aliases for these old symbols in the ABI linker script (missing_symbol = existing symbol;), and pass the script to a linker. After that missing symbols will appear in the library, and we are good - we made some changes in the library which caused ABI break, but thanks to such mechanism we avoided this ABI break.